### ISSUE TITLE
Correlation Id Token

### ISSUE THREAD

**Issue by stout01** [felixheck/laabr#2]
I am unable to get the correlation id token to display in my logs. Instead it is just outputting: `-`

`"laabr": "^2.2.2",`

My Config:
```javascript
    laabr.format('response', ':cid - :req[x-request-id] - :time[utc], :tags :req[host] :method :url :status (:responseTime)');

    server.register({
      register: laabr.plugin,
      options: {
        colored: true,
        correlator: {
          enabled: true,
          header: 'x-request-id'
        },
        pino: {
          level: 'trace'
        },
        hapiPino: {
          logPayload: true,
          mergeHapiLogData: true
        }
      }
    });
```

Logger Output:
`- - ThisIsTheRequestId - Tue, 18 Jul 2017 17:29:53 GMT, - localhost:22001 GET /408 200 (618)`

**Comment by felixheck** [comment]
Hey Andrew, thanks for your request.

First of all `options.hapiPino.mergeHapiLogData` does not work currently. It's disabled internally because the token `:message` needs the data nested in the `message` property of the logged message. If that's an additionally need, just push another issue or in best case a PR :) 

Next: is it possible that I take a look on your project?

**Comment by stout01** [comment]
I created a quick example app where I am still having the issue.

https://github.com/stout01/HapiLoggingExample

**Comment by felixheck** [comment]
![image](https://user-images.githubusercontent.com/14093205/28335615-4d7f2d02-6bff-11e7-99ee-82078efa4e43.png)

What's your npm/node version?

**Comment by stout01** [comment]
node: v8.1.1
npm: v5.3.0

**Comment by felixheck** [comment]
Installed node v8.1.1 & npm v5.3.0: works on macOS like a charm.
do you work w/ windows?

**Comment by felixheck** [comment]
![image](https://user-images.githubusercontent.com/14093205/28335843-0908694e-6c00-11e7-93e3-85c6e270bd37.png)

**Comment by stout01** [comment]
Your screen shot only shows the header once. The way I have the format line setup, it should show the `x-request-id` header twice in the log statement

**Comment by felixheck** [comment]
Nope, it shows the correlation id stored in the continuation-local-storage.
The dash in the middle is the request header field `x-request-id`. This is just truthy if you send an respective header:
![image](https://user-images.githubusercontent.com/14093205/28336210-59522c18-6c01-11e7-8aaf-1ff2cb080aa8.png)

![image](https://user-images.githubusercontent.com/14093205/28336121-1552458e-6c01-11e7-9e5b-65e47cdc3bbb.png)

**Comment by felixheck** [comment]
But yeah, you are right. There something wrong when using curl.
When using cocoa it works o.O

**Comment by felixheck** [comment]
It's necessary to send an additional `accept-encoding` header with the value/s `gzip` and/or `deflate`. Don't know why, gonna investigate.

**Comment by felixheck** [comment]
There's a release branch (https://github.com/felixheck/laabr/tree/release/2.2.3) containing the hotfix. would be nice if you could test it again in your environment but it should work :)

**Comment by stout01** [comment]
Perfect. Its working as expected now. Thanks for being so on top of it!

**Comment by felixheck** [comment]
Released as [v2.2.3](https://github.com/felixheck/laabr/releases/tag/v2.2.3)

Thanks for your issue!
BTW: I handled the `mergeHapiLogData` in this release as well.
